cookbooks/swift/templates/default/etc/swift/object-server/default.conf-template.erb

```diff
@@ -3,6 +3,8 @@ mount_check = false
 disable_fallocate = true
 workers = 1
 log_facility = LOG_LOCAL2
+# NOTE: servers_per_port necessary for replicator/reconstructor so must go here
+servers_per_port = <%= @servers_per_port %>
```

Comment:
I don't yet understand why replicator/reconstructor need this - I get why they might need bind_ip...